test/e2e/admin_credential_lifecycle.go

```diff
@@ -169,7 +174,7 @@ var _ = Describe("Customer", func() {
 				credentials = append(credentials, adminRESTConfig)
 
 				By(fmt.Sprintf("validating admin credential %d works", i+1))
-				Expect(verifiers.VerifyHCPCluster(ctx, adminRESTConfig)).To(Succeed())
+				Expect(verifiers.VerifyHCPCluster(ctx, adminRESTConfig, verifiers.VerifyBreakglassAdminAccess())).To(Succeed())
```

Comment:
Why add a verifier, will anyone else care to run this? IMO inlining it here makes more sense